src/main/java/seedu/address/logic/commands/ListEventsCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.AttendanceEvent;
+
+/**
+ * Lists all attendance events.
+ */
+public class ListEventsCommand extends Command {
+
+    public static final String COMMAND_WORD = "listevents";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all attendance events.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Attendance Events:\n%1$s";
+    public static final String MESSAGE_NO_EVENTS = "There are no attendance events to display.";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```

Comment:
Could there be more additions of definsive coding here? eg. try/catch, assertions, logger?